### PR TITLE
feat: add translation of `z.string().datetime()`

### PR DESCRIPTION
**Pull request by aiji42**
The new method `z.string().datetime()` has been added to zod v3.20.0-beta. It determines the input value is UTC datetimes.

Thank you in advance for your help in translating.

### CODE REVIEW

File: packages/core/locales/ar/zod.json [hunk 1]
```diff
@@ -72,7 +73,8 @@
     "url": "عنوان url",
     "uuid": "uuid",
     "cuid": "cuid",
-    "regex": "التعبير النمطي"
+    "regex": "التعبير النمطي",
+    "datetime": "datetime"
```
Comment (aiji42) [inline packages/core/locales/ar/zod.json:77]: @maherapp I would appreciate it if you could help me with the translation. here and line 22

File: packages/core/locales/fr/zod.json [hunk 2]
```diff
@@ -72,7 +73,8 @@
       "url": "lien",
       "uuid": "UUID",
       "cuid": "CUID",
-      "regex": "expression régulière"
+      "regex": "expression régulière",
+      "datetime": "datetime"
```
Comment (aiji42) [inline packages/core/locales/fr/zod.json:77]: @ismailajizou I would appreciate it if you could help me with the translation here and check line 22

File: packages/core/locales/zh-CN/zod.json [hunk 3]
```diff
@@ -72,7 +73,8 @@
     "url": "url",
     "uuid": "uuid",
     "cuid": "cuid",
-    "regex": "正则表达式"
+    "regex": "正则表达式",
+    "datetime": "datetime"
```
Comment (aiji42) [inline packages/core/locales/zh-CN/zod.json:77]: @izayoi-hibiki I would appreciate it if you could help me with the translation here and check line 22

File: packages/core/locales/ar/zod.json (hunk 1 above)
Comment (maherapp) [reply packages/core/locales/ar/zod.json:77]: Hello @aiji42, I think the line 22 is already translated.
`"datetime": "غير صالح {{validation}}",`
For the line 77, the translation would be:
`"datetime": "التاريخ والوقت"`

Comment (aiji42) [reply packages/core/locales/ar/zod.json:77]: @maherapp Thank you!
شكرًا لك

File: packages/core/locales/fr/zod.json (hunk 2 above)
Comment (aiji42) [reply packages/core/locales/fr/zod.json:77]: @ismailajizou Thank you! Merci!

**Pull request merged**